core/src/poh_recorder.rs

```diff
@@ -415,6 +415,8 @@ impl PohRecorder {
                     timing::duration_as_us(&now.elapsed()) as usize
                 );
                 if let Some(poh_entry) = res {
+                    inc_new_counter_info!("poh_recorder-num_entries", 1);
+                    inc_new_counter_info!("poh_recorder-num_transactions", transactions.len());
```

Comment:
We have other counters for entries and transactions, I don't think it will be significantly different than here.

Reply:
removed